### ISSUE TITLE
Support TOR and *.onion addresses

### ISSUE THREAD

**Issue by vn971** [funkyg/funkytunes#9]
By using Tor and *.onion addresses you could totally avoid any country-specific issues like the one raised here: https://github.com/funkyg/funkytunes/issues/7

It should optional, of course, as not all people have Orbot installed.

Thanks

**Comment by ghost** [comment]
This should only be used to fetch the torrent file, not for the real p2p traffic of course. TOR isn't meant for that.

**Comment by vn971** [comment]
Yes, of course.

**Comment by gjedeer** [comment]
While Onion support would be nice, you can route the regular domains with Orbot by going to settings and choosing "select apps" without support from the app.

**Comment by vn971** [comment]
@gjedeer regular domains can be blocked by the tor route you use.
What this issue proposes is to use *.onion addresses, which live within the TOR network and don't touch the common web.
I think there is a huge difference here in terms of maintenance.

**Comment by funkyg** [comment]
Looks like here is a onion address for pirate bay (not sure if official or mirror):
http://uj3wazyk5u4hnvtk.onion

It would require Orbot installed, and probably an option to enable/disable tor in the funkytunes settings. 

Also i dont think we can use tor for magnet download only, because we do both magnet and torrent download with jlibtorrent.

**Comment by vn971** [comment]
@funkyg yes, that's the correct one (onion address). I think it's official.

// I don't know about jlibtorrent. Could be a good feature request if it's not present, but well, IDK.
